lib/rubocop/cop/rspec/excessive_docstring_spacing.rb

```diff
@@ -52,14 +52,14 @@ def on_send(node)
 
         # @param text [String]
         def excessive_whitespace?(text)
-          return true if text.start_with?(' ') || text.end_with?(' ')
+          return true if text.start_with?(' ', '　') || text.end_with?(' ', '　')
 
           text.match?(/[^\n ]  +[^ ]/)
         end
 
         # @param text [String]
         def strip_excessive_whitespace(text)
-          text.strip.gsub(/  +/, ' ')
+          text.strip.gsub(/(\s|　)(\s|　)+/, ' ')
```

Comment:
I wonder if we could check if ‘strip’ped docstring matches raw one. And if ‘strip’ strips out em space. We could have taken care of more weird cases.

Reply:
It seems that `strip` does not strip em spaces. I updated this PR. What do you think?